sk0-front/src/main.ts

```diff
@@ -22,6 +24,8 @@ bootstrapApplication(AppComponent, {
     provideServiceWorker("ngsw-worker.js", {
       enabled: !isDevMode(),
       registrationStrategy: "registerWhenStable:30000"
-    })
+    }),
+    JwtHelperService,
+    { provide: JWT_OPTIONS, useValue: JWT_OPTIONS }
```

Comment:
Shouldn't we use the "old way" of using providers fir the bootstrap? Instead of manual providing the elements.

We can/should be able to use the 

```ts

importProvidersFrom([JwtModule.forRoot({})]),
```

or similar